### PR TITLE
Added Janapriyam TV

### PR DESCRIPTION
**Pull request by Niyasmkkv**
#4466

### CODE REVIEW

**Comment by Dum4G** [comment]
It's not playing for me? May be geo-blocked

**Comment by Niyasmkkv** [comment]
working fine for me on vlc.
image

File: channels/in.m3u [hunk 1]
```diff
@@ -247,6 +247,8 @@ http://103.199.161.254/Content/isaiaruvi/Live/Channel(IsaiAruvi)/index.m3u8
 http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p) [Not 24/7]
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
+#EXTINF:-1 tvg-id="JanapriyamTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/rbn6YSb.png" group-title="Entertainment",Janapriyam TV
```
Comment (Dum4G) [inline channels/in.m3u:250]:
```suggestion
#EXTINF:-1 tvg-id="JanapriyamTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/rbn6YSb.png" group-title="Entertainment",Janapriyam TV [Geo-blocked]
```

**Review by Dum4G — APPROVED** [review]

File: channels/in.m3u (hunk 1 above)
Comment (freearhey) [inline channels/in.m3u]: Return "HTTP 451 Unavailable For Legal Reasons" if you try to run it outside of India.

```suggestion
#EXTINF:-1 tvg-id="JanapriyamTV.in" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/rbn6YSb.png" group-title="Entertainment",Janapriyam TV [Geo-blocked]
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Pull request merged**